src/main.cpp

```diff
@@ -36,6 +36,8 @@
 #include "ReqDocumentPdf.h"
 #include "renderingHtml.h"
 
+extern std::string htmlcss;
+
```

Comment:
This declaration would better be in req.h, with a comment explaining its purpose (eg: "css file for html output").

Reply:
OK